### DISCUSSION TITLE
DataCite Commons: Consider including contributions under "works" associated with an ORCID

### DISCUSSION THREAD

**Discussion by imathews** [datacite/datacite-suggestions#117] [General Suggestions]
### What is the problem that your suggestion solves?

We have a lot of scenarios where, e.g., a librarian will invest a lot of effort in curating a dataset to make it research available. While they aren't the creator of the dataset, their contributions are often significant, and I'm sure we'd all want them to receive appropriate credit.

Right now, when an individual contributor (with ORCID) is recorded in the DataCite metadata, that contribution doesn't show up under that person's works.

To provide an example:
- [This dataset on Redivis](https://redivis.com/datasets/b29g-apdxfe01g) lists Kate Barron as a contributor.
- This is [reflected in the DataCite Commons record](https://commons.datacite.org/doi.org/10.57761/1s63-x006) for this dataset, under "Contributors".
- However, if we go to the [record associated with Kate's ORCID in DataCite](https://commons.datacite.org/orcid.org/0000-0003-2538-7582), we don't see any "Works" listed, which I feel dramatically undersells Kate's contributions to the community!

### What solution might meet your needs?

It would be great to see contributions take greater prominence in DataCite Commons. The overarching goal, as I see it, is to ensure that research outputs credit everyone who made that output possible. Too often, dataset curators go unrecognized, which in the long run disincentivizes such curation, to the detriment of the community. 

There's probably a discussion to be had around what level of contribution "counts" for a work to be listed. I think my initial inclination is to just include everyone equally, though future iterations of the metadata specification could explore how to record the magnitude of a given contribution.

### Your name

Ian Mathews

### Your organization

Redivis

### Is there anything else you would like to share?

Potentially related: https://github.com/datacite/datacite-suggestions/discussions/43

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by zchandler** [comment]
I think this issue would address a major gap we have in providing recognition for data curators. Subscribing!

**Comment by zchandler** [comment]
One thing that we should agree on as a community is how to make this "Work" in ORCID, the work of the data curator who makes a given dataset substantially more useful, explicit and unambiguous as related to, but not the same as, creation of the dataset itself.  Something like a "data enhancement" or "data curation" work. So there could be a widely known dataset whose source is a bit messy, but still deserves the ur designation as dataset source, and the Redivis version could get a separate DOI with the "sameAs" or "derivedFrom" relationship to the original. Then the local data curator could be recognized for their improvements to the downstream version (only).

**Comment by codycooperross** [comment]
Thank you for the suggestions and thoughts, @imathews and @zchandler ! This is aligned with suggestions around [supporting CRediT roles](https://credit.niso.org/) and enhancing support for contributors in the Schema:

- https://github.com/datacite/datacite-suggestions/discussions/32 

We've also implemented charts in Commons to visualize how contributors relate to the outputs of a project. [See an example here](https://commons.datacite.org/doi.org/10.60581/zaev-6p15) under "Related Works."

We will watch this thread for further discussion, so feel free to provide additional context and related suggestions here.

**Comment by zchandler** [comment]
Thanks @codycooperross !  I think we have an opportunity to do some good here, because there are research data systems, like the one that @imathews provides, that have a record of important contributions by data curators that the PIs and research teams don't know about. All researchers see is a spiffy, cleaned L2 dataset in Redivis, but all the work that went into cleaning it generally does not (ever) make it to the author line of a journal article. They are much more likely to credit the data specialist postdoc in their lab (as they should), but they never meet the data librarian that got them their start.  I think through DataCite DOIs and ORCID records we have a shot at elevating these contributions, even if they don't meet the ICMJE bar for authorship.

**Comment by codycooperross** [comment]
@imathews People pages in Commons now display works where the person is either a creator or a contributor. The work you mentioned [now appears in Kate's People page in Commons](https://commons.datacite.org/orcid.org/0000-0003-2538-7582). People pages are also more consistently linked on Works pages when creators or contributors have an ORCID nameIdentifier in their metadata. 

These changes only start to address some of the larger discussions in this thread, but I hope it helps! Let us know if you have further feedback.

**Comment by imathews** [comment]
Thanks @codycooperross – this is really great to see! Agree with @zchandler that this is all likely part of a broader conversation around works and their presence in ORCID, but it's so helpful to have a concrete example of this all in DataCite Commons. 

Thanks again for all your efforts and follow up here!

**Reply by KellyStathis** [reply]
Relating to this broader conversation around works and their presence in ORCID, we're now working on scoping a change to DataCite's ORCID auto-update to include Contributors. I've added a comment to this thread: https://github.com/datacite/datacite-suggestions/discussions/249 with what's currently being considered and an open question we have. It would be great to get your perspective, @imathews and @zchandler. Thanks for all your thinking on this so far!
